core/src/main/java/org/elasticsearch/plugins/InstallPluginCommand.java

```diff
@@ -193,17 +193,17 @@ protected void execute(Terminal terminal, OptionSet options, Environment env) th
     }
 
     // pkg private for testing
-    void execute(Terminal terminal, String pluginId, boolean isBatch, Environment env) throws Exception {
-        if (pluginId == null) {
-            throw new UserException(ExitCodes.USAGE, "plugin id is required");
+    void execute(Terminal terminal, String pluginName, boolean isBatch, Environment env) throws Exception {
```

Comment:
It is called pluginId in install because it is an identifier, which _may_ be a plugin name, but it also may be maven coordinates or a url.